plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -2057,3 +2097,488 @@ describe('PersonState.update()', () => {
         })
     })
 })
+
+describe('person id overrides', () => {
+    // For some reason these tests failed if I ran them with a hub shared
+    // with other tests, so I'm creating a new hub for each test.
+    let hub: Hub
+    let closeHub: () => Promise<void>
+    let teamId: number
+    const timestamp = DateTime.fromISO('2020-01-01T12:00:05.200Z').toUTC()
+
+    beforeAll(async () => {
+        ;[hub, closeHub] = await createHub({})
+    })
+
+    beforeEach(async () => {
+        const organizationId = await createOrganization(hub.db.postgres)
+        teamId = await createTeam(hub.db.postgres, organizationId)
+        // jest.useFakeTimers({ advanceTimers: 50 })
```

Comment:
^

Reply:
No idea what "^" means. Delete this comment?

Reply:
"^" here meant you probably don't want to merge like this, can you figure out how this test should be fixed or if the comment should be removed or I don't know exactly what needs to happen, but we don't want to merge commented out code generally - it was here because it was WIP initially I believe

Reply:
I'm fine with deleting the comment, test is passing so I don't think there's anything to fix.